test/tests/functional/pbs_nonprint_characters.py

```diff
@@ -606,13 +606,17 @@ def test_nonprint_character_in_qstat_json_valid(self):
         qsub -v "var1='A,B,<non-printable character>,C,D'"
         and check that the qstat -f -F json output is valid
         """
+        uhost = PbsUser.get_user(TEST_USER).host
         self.npch_exclude += ['\x1C']
         for ch in self.npcat:
             self.logger.info('##### non-printable char: %s #####' % repr(ch))
             if ch in self.npch_exclude:
                 self.logger.info('##### excluded char: %s' % repr(ch))
                 continue
-            a = {ATTR_v: r"var1=\'A\,B\,%s\,C\,D\'" % ch}
+            if uhost is None or self.du.is_localhost(uhost):
```

Comment:
U don't need `uhost is None` check, as `is_localhost` will return True is uhost is None